packages/@react-spectrum/utils/src/BreakpointProvider.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {ReactNode, useContext, useEffect, useState} from 'react';
+import {useIsSSR} from '@react-aria/ssr';
+
+interface Breakpoints {
+  S?: number,
+  M?: number,
+  L?: number,
+  [custom: string]: number
+}
+
+interface IBreakpointContext {
+  matchedBreakpoints: string[]
+}
+
+export const BreakpointContext = React.createContext<IBreakpointContext>(null);
+BreakpointContext.displayName = 'BreakpointContext';
+
+// This is temporary values.
+export const DEFAULT_BREAKPOINTS = {S: 380, M: 768, L: 1024};
```

Comment:
Do all of the exports in the file need to be here? Looks like DEFAULT_BREAKPOINTS is only used in Provider?

Reply:
I can remove a few of them, should we move this one to just live in the Provider instead of in BreakpointProvider?

Reply:
yeah let's put it in Provider and we can move it if we need it elsewhere